src/main/java/com/hivemq/util/ObjectMemoryEstimation.java

```diff
@@ -22,13 +22,16 @@
  */
 public class ObjectMemoryEstimation {
 
-    public static int ENUM_OVERHEAD = 24;
-    public static int STRING_OVERHEAD = 38;
-    public static int ARRAY_OVERHEAD = 12;
-    public static int LONG_WRAPPER_SIZE = 16;
-    public static int INT_WRAPPER_SIZE = 16;
-    public static int LONG_SIZE = 8;
-    public static int INT_SIZE = 4;
+    public static final int OBJECT_SHELL_SIZE   = 8;
```

Comment:
what is the "object shell size"? If this is the overhead of an object, then the value is 12 bytes